sklearn/tests/test_public_functions.py

```diff
@@ -275,10 +275,6 @@ def _check_function_param_validation(
     "sklearn.metrics.pairwise.linear_kernel",
     "sklearn.metrics.pairwise.manhattan_distances",
     "sklearn.metrics.pairwise.nan_euclidean_distances",
-    "sklearn.metrics.pairwise.paired_cosine_distances",
```

Comment:
I would still test them but add a comment above to remove them.

Reply:
I think this still needs to be reverted.